providers/implementations/keymgmt/dh_kmgmt.c

```diff
@@ -476,6 +477,10 @@ static int dh_gen_set_params(void *genctx, const OSSL_PARAM params[])
         if (gctx->md == NULL)
             return 0;
     }
+    p = OSSL_PARAM_locate_const(params, OSSL_PKEY_PARAM_DH_PRIV_LEN);
+    if (p != NULL
```

Comment:
Does this fit on one line?

Reply:
yes - will change - will aplly it to the rest of the function also if that is preferred